src/main/java/net/glowstone/block/itemtype/ItemType.java

```diff
@@ -164,10 +166,67 @@ public final String toString() {
         return getClass().getSimpleName() + "{" + getId() + " -> " + getMaterial() + "}";
     }
     
+    //Code for wearable items
+    private ItemWearablePosition wearablePosition = ItemWearablePosition.NON_WEARABLE;
+    private int armorPoints = 0;
+    private boolean fastEquip = false;
+    
     public ItemWearablePosition getWearablePosition() {
         return wearablePosition;
     }
     public void setWearablePosition(ItemWearablePosition position) {
```

Comment:
Then use an interface with default methods for now until the new types system comes around, which will fix these inheritance issues.